apps/cnspec/cmd/scan.go

```diff
@@ -388,6 +415,27 @@ This example connects to Microsoft 365 using the PKCS #12 formatted certificate:
 	},
 })
 
+func askToUploadReport() bool {
+	reader := bufio.NewReader(os.Stdin)
```

Comment:
This should move to a cli component. If it gets more complex also consider using https://github.com/charmbracelet/bubbletea/tree/master/examples#result